.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
- [ ] I would recommend to avoid pushing the `.env` file on Github. Most of the time you will be having the sensitive data in the `.env` file. so instead of pushing, just put in the `.gitignore` file.

Reply:
Thank you for suggesting what to do. I did it. I put .env in .gitignore file.